package.json

```diff
@@ -427,6 +427,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "%asciidoc.useWorkspaceRoot.desc%"
+				},
+				"asciidoc.antora.enableAntoraSupport": {
+					"type": "boolean",
+					"default": false,
+					"description": "%asciidoc.antora.desc%"
```

Comment:
you need to add a translation in `package.nls.json` to explain what this setting does